packages/main/src/Popover.js

```diff
@@ -197,202 +233,209 @@ class Popover extends Popup {
 		return litRender;
 	}
 
+	static get styles() {
+		return PopoverCss;
+	}
+
 	static get template() {
 		return PopoverTemplate;
 	}
 
-	static get styles() {
-		return [Popup.styles, popoverCss];
+	static async define(...params) {
```

Comment:
If there is no additional logic in the define, you can remove the entire method (the UI5Element`s one would be called)